### ISSUE TITLE
Image header is ignored

### ISSUE THREAD

**Issue by smoh** [lgrcia/prose#171]
I think the `header` keyword to `Image` instance is being ignored because of [this line](https://github.com/lgrcia/prose/blob/eedfa9f19773333f344206f2bceec2ced398c279/prose/core/image.py#L75).

```python
import prose
print(prose.__version__)
from prose import Image
import numpy as np

d = np.ones((512,512))
hdr = dict(FILTER='B')

print(Image(d, header=hdr).header)
# hack to get around the bug
print(Image(d, computed=(dict(fits_header=hdr))).header)
```

output:
3.3.1
END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
{'FILTER': 'B'}

**Comment by lgrcia** [comment]
Thanks for opening this issue @smoh! I opened and merged a pull request to fix it. It is now part of the patched version `3.3.2`, published on pypi.

**Comment by smoh** [comment]
Thank you! I will check it out.
